php/utils.php

```diff
@@ -1406,7 +1406,7 @@ function esc_sql_ident( $idents ) {
  * @return bool Whether the provided string is a valid JSON representation.
  */
 function is_json( $argument, $ignore_scalars = true ) {
-	if ( empty( $argument ) || ! is_string( $argument ) ) {
+	if ( ! is_string( $argument ) || '' ===  $argument ) {
```

Comment:
Removing the extra space after the equals will satisfy the wpcs sniff, so `'' ===  $argument` goes to `'' === $argument` (hard to see on github!).

Reply:
Oh! Sorry I didn't notice it.